app/controllers/settings/reference_payment_controller.rb

```diff
@@ -26,6 +24,14 @@ def create
 
   def assign_form_objects
     @reference_payment = ReferencePaymentSettings.new(service_id: service.service_id)
+    @confirmation_email_enabled = confirmation_email_setting('dev').try(:send_confirmation_email?) && confirmation_email_setting('production').try(:send_confirmation_email?)
+  end
+
+  def confirmation_email_setting(environment)
+    SubmissionSetting.find_by(
```

Comment:
`find_by` will return the first record it finds. would `where` be more appropriate, to return a relation of both environments?

Reply:
never mind me, you're calling it twice 😆